smartsim/_core/launcher/local/local.py

```diff
@@ -60,10 +56,11 @@ def create_step(self, name: str, cwd: str, step_settings: SettingsBase) -> Step:
             raise TypeError(
                 f"Local Launcher only supports entities with RunSettings, not {type(step_settings)}"
             )
-        step = LocalStep(name, cwd, step_settings)
-        return step
+        return LocalStep(name, cwd, step_settings)
 
-    def get_step_update(self, step_names: t.List[str]) -> t.List[t.Tuple[str, t.Optional[StepInfo]]]:
+    def get_step_update(
+        self, step_names: t.List[str]
+    ) -> t.List[t.Tuple[str, t.Optional[StepInfo]]]:
```

Comment:
Re-synced the typehint and docstring!

Looks the de-sync was coming because docstring was _technically_ "more correct" but we were expecting a `list` return type which is invariant and thus breaking liskov's substitution principle. Might be worth going through these methods and updating them with the co-variant equivalents (not part of this ticket)!